app/models/pacbio/run.rb

```diff
@@ -12,7 +12,7 @@ class Run < ApplicationRecord
     # Sequel II and Sequel I are now deprecated
     enum system_name: { 'Sequel II' => 0, 'Sequel I' => 1, 'Sequel IIe' => 2, 'Revio' => 3 }
 
-    after_create :generate_name
+    after_create :generate_name, :generate_comment
```

Comment:
I think it might make sense to do this as a before_create so we don't have to do 2 saves. Sorry my fault didn't think aboout this.